packages/xo-web/src/xo-app/vm/tab-advanced.js

```diff
@@ -780,6 +784,13 @@ export default class TabAdvanced extends Component {
                   labelId='vmWarmMigration'
                   tooltip={isWarmMigrationAvailable ? undefined : _('availableXoaPremium')}
                 />
+                <TabButton
+                  btnStyle='warning'
+                  handler={isMigrationAllowed ? vmBlockMigration : vmAllowMigration}
+                  handlerParam={vm}
+                  icon='vm-allow-migration'
+                  labelId={isMigrationAllowed ? 'vmBlockMigration' : 'vmAllowMigration'}
+                />
```

Comment:
It seems I lost a comment there: use a toggle button like other blocked operations.

What I previously meant is: do not create a function that will switch the state as it can behave strangely if the cache is not up-to-date, the customer should be able to see whether it will block or unblock when clicking on the button.

Reply:
That's what I did: I split my toogle function into two different functions, and the button indicates whether it is used to block or unblock (as shown on the screenshots at the top of the PR).
What's wrong for you?

Reply:
I want this feature to be displayed the same way other blocked operations are displayed: 
![image](https://github.com/user-attachments/assets/53416b12-165c-4dd6-b6b8-29033ca4f91a)

Reply:
@julien-f OK it's clearer with a picture! I'll change that.

Reply:
Done